### PR TITLE
UT_Alignment_12d_3

### PR DESCRIPTION
**Pull request by LeeGregory12d**
# IFC4.3 Unit test
Simple alignment with left arc HG and crest parabola VG - semantics only

## Review requested
- IfcAlignmentVerticalSegment - PredefinedType PARABOLICARC for crest
- 
@pjanck

### CODE REVIEW

**Comment by larswik** [comment]
Right now it seems as if the uploaded file contains things like "<<<<<<< Updated upstream" etc. Can you please upload the alignment file?

**Comment by LeeGregory12d** [comment]
Sorry about that - I will upload the unadulterated file

File: IFC 4.3/UT_Alignment_12d_3/readme.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+
+# UT_Alignment_12d_3
+# Simple alignment with left arc in horizontal geometry and crest parabola for vertical geometry
+# There is only the semantic definition and no ifc geometry 
+
+### Intent
+
+IFC 4x3 RC2
+This scenario has a simple alignment consisting of a:
+<<<<<<< Updated upstream
```
Comment (larswik) [inline IFC 4.3/UT_Alignment_12d_3/readme.md:10]: It seems like the readme-file needs to be re-uploaded. Same issue as with the IFC files.

**Comment by larswik** [comment]
IFC file ok, but update the readme.

**Review by larswik — COMMENTED** [review]
IFC file ok but update readme

**Review by larswik — APPROVED** [review]

**Comment by LeeGregory12d** [comment]
UT moved to repository IFC-infra-unit-test

**Pull request closed without merging**